bin/submission_new.py

```diff
@@ -533,15 +544,26 @@ def add_action_block(self, submission):
 		file1 = ET.SubElement(add_files, "File", file_path=self.sample.fastq1)
 		data_type1 = ET.SubElement(file1, "DataType")
 		data_type1.text = "generic-data"
-		file2 = ET.SubElement(add_files, "File", file_path=self.sample.fastq2)
-		data_type2 = ET.SubElement(file2, "DataType")
-		data_type2.text = "generic-data"
```

Comment:
I think we need these lines....they are the R2 fastq files

Reply:
Added this back in.